ctsvi_ad_node:
  ros__parameters:
    q_init: 0.1
    timestep: 0.05
    duration: 5.0
    urdf_path: /home/space/ros2_ws/dynamic_ws/src/vi/urdf/7_pendulum.urdf
